lib/node_modules/@stdlib/stats/base/dists/uniform/ctor/benchmark/benchmark.js

```diff
@@ -92,7 +92,7 @@ bench( pkg+'::set:a', function benchmark( bm ) {
 
 	bm.tic();
 	for ( i = 0; i < bm.iterations; i++ ) {
-		y = ( 100.0*randu() ) + EPS;
+		y = uniform( EPS, 100.0);
```

Comment:
```suggestion
		y = uniform( EPS, 100.0 );
```

This change should be applied consistently throughout the PR.